common/app/com/commercetools/sunrise/ctp/graphql/GraphQLRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.commercetools.sunrise.ctp.graphql;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.client.SphereRequest;
+
+import javax.annotation.Nullable;
+
+public interface GraphQLRequest<T> extends SphereRequest<T> {
```

Comment:
I think for now it's a good idea to extend from `SphereRequest`, but I don't know if the type parameter `T`will make sense for GraphQL in the future. But I really like that we now use our GraphQL API in sunrise 👍 and it will give us more insights on how to support it in our SDK 😉